Annotato/Annotato/Persistence/PersistenceService/RemotePersistenceService/RemoteAnnotationsPersistence.swift

```diff
@@ -1,7 +1,7 @@
 import AnnotatoSharedLibrary
 
-struct RemoteAnnotationsPersistence: AnnotationsPersistence {
-    func createAnnotation(annotation: Annotation) async -> Annotation? {
+struct RemoteAnnotationsPersistence: AnnotationsRemotePersistence {
+    func createAnnotation(annotation: Annotation, webSocketManager: WebSocketManager?) async -> Annotation? {
```

Comment:
Instead of passing the websockets as params, would it be better if we make it an instance property? Same for `RemoteDocumentsPersistence`